testutils/localstack.go

```diff
@@ -0,0 +1,55 @@
+package testutils
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Layr-Labs/eigensdk-go/aws"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/kms/types"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const LocalStackPort = "4566"
+
+func StartLocalstackContainer(name string) (testcontainers.Container, error) {
+	fmt.Println("Starting Localstack container")
+	req := testcontainers.ContainerRequest{
+		Image:        "localstack/localstack:latest",
+		Name:         fmt.Sprintf("localstack-test-%s", name),
+		Env:          map[string]string{"GATEWAY_LISTEN": fmt.Sprintf("0.0.0.0:%s", LocalStackPort), "LOCALSTACK_HOST": fmt.Sprintf("localhost.localstack.cloud:%s", LocalStackPort)},
```

Comment:
not sure you want to set this GATEWAY_LISTEN like this. Doesn't really matter since its just a short lived test, but this will expose your localstack instance to the internet. default will only listen on localhost, which should be enough.

Reply:
setting it like you do does this:
<img width="1128" alt="image" src="https://github.com/Layr-Labs/eigensdk-go/assets/9342524/28c39e72-b981-4889-b29e-f6ef2a6dfa68">

Reply:
whereas the default (not specifying GATEWAY_LISTEN) is
<img width="1125" alt="image" src="https://github.com/Layr-Labs/eigensdk-go/assets/9342524/67756313-c420-4d2e-b2c0-513cd985c12b">
which should be sufficient I think... at least that's how I've been using it

Reply:
Removed GATEWAY_LISTEN and updated endpoint to 127.0.0.1